src/WebJobs.Extensions.Http/Config/HttpConfiguration.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// Defines the configuration options for the Http binding.
+    /// </summary>
+    public class HttpConfiguration : IExtensionConfigProvider
```

Comment:
>HttpConfiguration [](start = 17, length = 17)

Can we give this a different name so that it doesn't conflict with System.Web.Http.HttpConfiguration?